include/zephyr/net/net_if.h

```diff
@@ -2562,6 +2562,15 @@ bool net_if_ipv4_set_netmask_by_addr(struct net_if *iface,
 				     const struct in_addr *addr,
```

Comment:
Please change the commit topic to `net: if: Add net_if_ipv4_get_gw helper function`
Also you need to add a commit body, write there why this is needed.

Reply:
@jukkar Updated